implementation.md

```diff
@@ -259,6 +262,13 @@ All plugin integration is provided by:
 - adding annotated methods in `PluginFile`
 - using the `Definer`
 
+A number of plugin features can be added to the `PluginFileType` or the
+`PluginFile`. Anything placed in the `PluginFileType` will be a global
+defintion. The Shesmu adminstrator can create configuration files that will
```

Comment:
This makes it sound a little like the admin is purposefully defining 'PluginFile' instances somewhere in the config files. Every configuration file automatically spawns an instance of (a child of) PluginFile, right?